homeassistant/components/lidarr/sensor.py

```diff
@@ -41,9 +42,8 @@ def get_modified_description(
 ) -> tuple[LidarrSensorEntityDescription[T], str]:
     """Return modified description and folder name."""
     desc = deepcopy(description)
-    name = mount.path.rsplit("/")[-1].rsplit("\\")[-1]
+    name = os.path.basename(os.path.normpath(mount.path)).lower()
     desc.key = f"{description.key}_{name}"
-    desc.name = f"{description.name} {name}".capitalize()
```

Comment:
Why do we remove the name? I also see that there is a translation key but why?

Reply:
Well it looks like I added translations key and not you. What change do you recommend?

Reply:
I am guessing you have lidarr set up locally, how does the entity look like? My guess is that it's ignoring translation key, and takes the name given from this function, but since `description.name` is not set, I expect it to look strange, can you confirm this?

Reply:
Sure.

Follow up question for this change:

Do we need a unique_id migration? Does the `os` functions reply the same on every platform?